migtests/scripts/functions.sh

```diff
@@ -893,7 +893,6 @@ normalize_json() {
             .OptimalInsertConnectionsPerNode? = "IGNORED" |
             .RowCount? = "IGNORED" |
 			.FeatureDescription? = "IGNORED" | # Ignore FeatureDescription instead of fixing it in all tests since it will be removed soon
-			.AssessmentIssues? = "IGNORED" | # Ignore AssessmentIssues until all tests are updated
```

Comment:
PR title says cleanup of report struct but I don't see any code changes. Is that not included in this PR?

Reply:
Updated PR desc and jira ticket also